addons/forum-post-countdown/addon.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Cooldown display for posting in the forums",
+  "description": "After submitting a forum post, display a countdown showing when you can post again.",
+  "tags": ["community", "forums"],
+  "versionAdded": "1.33.0",
+  "userscripts": [
+    {
+      "url": "post-create.js",
```

Comment:
Since this runs on the same pages as the main script could it be merged with it?